src/main/webapp/app/quiz/participate/quiz-scoring-infostudent-modal/quiz-scoring-info-student-modal.component.ts

```diff
@@ -0,0 +1,182 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import { Question, QuestionType, ScoringType } from 'app/entities/question';
+import { DragAndDropMapping } from '../../../entities/drag-and-drop-mapping';
+import { AnswerOption } from '../../../entities/answer-option';
+import { MultipleChoiceQuestion } from '../../../entities/multiple-choice-question';
+import { DragAndDropQuestion } from '../../../entities/drag-and-drop-question';
+import { ShortAnswerQuestion } from '../../../entities/short-answer-question';
+import { ShortAnswerSubmittedText } from 'app/entities/short-answer-submitted-text';
+import { TranslateService } from '@ngx-translate/core';
+
+@Component({
+    selector: 'jhi-quiz-scoring-infostudent-modal',
+    templateUrl: './quiz-scoring-info-student-modal.component.html',
+    styles: []
+})
+export class QuizScoringInfoStudentModalComponent implements OnInit {
+    readonly DRAG_AND_DROP = QuestionType.DRAG_AND_DROP;
+    readonly MULTIPLE_CHOICE = QuestionType.MULTIPLE_CHOICE;
+    readonly SHORT_ANSWER = QuestionType.SHORT_ANSWER;
+
+    readonly ALL_OR_NOTHING = ScoringType.ALL_OR_NOTHING;
+    readonly PROPORTIONAL_WITH_PENALTY = ScoringType.PROPORTIONAL_WITH_PENALTY;
+
+    @Input() score: number; // Score of the student that has been achieved
+    @Input() questionIndex: number; //Question Index of the question
+    @Input() question: Question;
+    @Input() DragAndDropMapping = new Array<DragAndDropMapping>();
+    @Input() MultipleChoiceMapping = new Array<AnswerOption>();
+    @Input() ShortAnswerText = new Array<ShortAnswerSubmittedText>();
+
+    /* Multiple Choice Counting Variables*/
+    multipleChoiceCorrectAnswerCorrectlyChosen: number; //Amount of right options chosen by the student
+    multipleChoiceWrongAnswerChosen: number; //Amount of wrong options chosen by the student
+    amountOfCorrectMultipleChoiceAnswers:number; //Amount of correct options for the question
```

Comment:
forgotten space
also be consistent when it comes to numbers: I would leave out 'amount' in all variables